VideoLocker/res/drawable/circular_semi_transparent_gray_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this should be named "semi_transparent_gray_circle" instead of specifying "background", since we aren't specifying where it's being used.